lib/pyspark_init.py

```diff
@@ -31,6 +31,14 @@ def load_data(spark: SparkSession) -> DataFrame:
         .option("delimiter", "\t")
         .csv("title.principals.csv")
     )
+    
+    for column in ['primaryProfession', 'knownForTitles']:
+       df_name_basics = df_name_basics.withColumn(column, when(df_name_basics[column] == "\\N", None).otherwise(df_name_basics[column]))
+    for column in ['titleType', 'primaryTitle', 'originalTitle', 'genres']:
+      df_title_basics = df_title_basics.withColumn(column, when(df_title_basics[column] == "\\N", None).otherwise(df_title_basics[column]))
+    for column in ['ordering', 'category', 'job', 'characters']:
+      df_title_principals = df_title_principals.withColumn(column, when(df_title_principals[column] == "\\N", None).otherwise(df_title_principals[column])
```

Comment:
Brakuje mi niektórych kolumn, których używamy później -  tconst, startYear, endYear, isAdult z df_title_basics, tconst, nconst z df_title_principals, nconst, primaryName z df_name_basics. Niektórych kolumn natomiast nie używamy - primaryTitle, job, primaryProfession. Czy mogłabyś dodać wszystkie te kolumny i usunąć niepotrzebne, chyba że uważasz którąś z nich za niepotrzebną do dodania tutaj, to daj komentarz proszę.

Reply:
kolumny wymienione jako brakujące nie wymagają zamiany "\\N" na null, zbędne już usunięte

Reply:
to mam pytanie, żeby się upewnić - w kodzie chyba zamieniamy na None prawda? W takim razie chcemy zamienić na None czy na null? Zadaje pytanie, bo dobrze byłoby rozważyć, co by było lepsze.

Reply:
Tak, zmieniamy na None,
None to null w pysparku i wyświetla się w tabeli jako null